src/Task.ts

```diff
@@ -110,6 +110,8 @@ export class Task {
     // description: '#dog #car http://www/ddd#ere #house'
     // matches: #dog, #car, #house
     public static readonly hashTags = /(^|\s)#[^ !@#$%^&*(),.?":{}|<>]*/g;
+    public static readonly subTagRegex =
+        /(#[^/\s]+)(\/[^ !@#$%^&*(),.?":{}|<>]+)/u;
```

Comment:
The previous regex, hashTags, has  a comment with examples, saying what it would match. That would be helpful for this one too.

Reply:
For sure! Great point. Adding...